### DISCUSSION TITLE
How can I help? (not a professional developer)

### DISCUSSION THREAD

**Discussion by perigvennetier** [boyan-soubachov/tastyworks_api#59] [General]
Hi all,

I'd love to use this eventually when I understand it better and it has the features I'd like to use (find a symbol, get the live price, get option prices and greeks, trade on specific mechanical data from this information).

In the meantime, I'd love to participate in the effort of bring this up to date if needed, helping with low hanging stuff and do what I can to move this project forward.

**Comment by bandwiches** [comment]
That's up to you, figure out what you want to do and plug it in where you can.  Sometimes I just sift through the issues for some low hanging (but challenging) fruit to see if I can't help there - even if it's just documentation or typo's.  Documentation is a great place to start cause it will force you to follow the code and learn what it's actually doing, then you can branch out from there.  I get a lot of ideas when I go over this kind a stuff, even if it is mundane on the outside.

**Reply by perigvennetier** [reply]
I’ll start checking it out when I have a few minutes. I have to set myself up with GitHub properly first ;)

**Reply by bandwiches** [reply]
@perigvennetier you could check out "issues" for something to start with if you're still looking!

I was wrong - I meant to say the Project board!
